utilities_common/pfc_stat_history.py

```diff
@@ -0,0 +1,209 @@
+import sys
```

Comment:
can you add new command document to CLI guide doc? [https://github.com/sonic-net/sonic-utilities/blob/master/doc/Command-Reference.md](https://github.com/sonic-net/sonic-utilities/blob/master/doc/Command-Reference.md)